src/tables.ts

```diff
@@ -79,8 +79,9 @@ export const CommonMetrics: Record<string, any> = {
   },
 
   StaveConnector: {
-    fontFamily: 'Times New Roman, serif',
-    fontSize: 16,
+    text: {
+      fontSize: 16,
+    },
```

Comment:
I don't understand why this fontSize is under a text { } sub element, while below on line 109, the fontSize is declared directly on the StaveText element.

Reply:
StaveText is text only, as a consequence, no distinction is required. StaveConnector consist oh glyphs and text.